### DISCUSSION TITLE
Profile for Raise3D Pro3 with Hyperspeed Upgrade Kit

### DISCUSSION THREAD

**Discussion by mcavuto** [OrcaSlicer/OrcaSlicer#2154] [General]
Would it be possible for someone to create a printer profile for the Raise3D Pro3 that takes advantage of the Hyperspeed Upgrade Kit? It enables speeds somewhere between the original Pro3 and the X1/P1P.  Thank you in advance if anyone is open to doing this!

**Comment by StevenTheAnimal** [comment]
Would like the same for the Pro3 Plus, with hyperspeed kit
